src/test/suite/fixtures/recorded/languages/elm/changeKey.yml

```diff
@@ -0,0 +1,30 @@
+languageId: elm
```

Comment:
This one should actually be "name".  You could prob also support "key" if you'd like, but "name" is more typical when the thing after the `:` is a type